content/en/platform/corda/1.6/cenm/release-notes.md

```diff
@@ -26,9 +26,37 @@ title: Release notes
 ### Upgraded dependencies
 
 This release includes the following dependency version changes between CENM 1.5.9 and 1.6:
-* H2 has been upgraded from 1.4.197 to 2.1.214.
-* Hibernate has been upgraded from 5.4.32.Final to 5.6.14.Final.
-* Hibernate Validator has been upgraded to 6.2.5.Final.
-* Liquibase has been upgraded from 3.6.3 to 4.20.0.
-* Apache Tomcat has been upgraded from 9.0.80 to 9.0.81.
-* Bouncy Castle has been upgraded from 1.68 to 1.75.
+* Hibernate has been upgraded from 5.4.32.Final to 5.6.14.Final
+* Hibernate Validator has been upgraded to 6.2.5.Final
+* Apache Tomcat has been upgraded from 9.0.80 to 9.0.81
+* Bouncy Castle has been upgraded from 1.68 to 1.75
+
+* H2 has been upgraded from 1.4.197 to 2.1.214
```

Comment:
This should be changed to 2.2.214 as well (I haven't actually updated this yet, that is the corda version but the aim is to update to this version for cenm 1.6 as well)